src/main/java/uk/gov/hmcts/et/common/model/ccd/types/RespondentTse.java

```diff
@@ -21,6 +21,4 @@ public class RespondentTse {
     private String copyToOtherPartyYesOrNo;
     @JsonProperty("copyToOtherPartyText")
     private String copyToOtherPartyText;
-    @JsonProperty("storedApplicationId")
-    private String storedApplicationId;
```

Comment:
I don't think this and the one below should be deleted? Think it's what @cindychmcts is working on